app/views/contexts/section_preview_context.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Mapping, Optional, Union
+
+from app.data_models import AnswerStore, ListStore, ProgressStore
+from app.data_models.metadata_proxy import MetadataProxy
+from app.questionnaire import QuestionnaireSchema
+from app.views.contexts.context import Context
+from app.views.contexts.preview import PreviewGroup
+
+
+class SectionPreviewContext(Context):
+    def __init__(
+        self,
+        *,
+        language: str,
+        schema: QuestionnaireSchema,
+        answer_store: AnswerStore,
+        list_store: ListStore,
+        progress_store: ProgressStore,
+        metadata: Optional[MetadataProxy],
+        response_metadata: Mapping[str, Union[str, int, list]],
+        section_id: str,
+    ):
+        super().__init__(
+            language,
+            schema,
+            answer_store,
+            list_store,
+            progress_store,
+            metadata,
+            response_metadata,
+        )
+        self._section_id = section_id
+
+    def __call__(self) -> dict[str, dict]:
+        preview = self._build_preview()
+
+        return {
+            "preview": {
+                **preview,
+            }
+        }
```

Comment:
Can return directly.
```suggestion
        return {
            "preview": self._build_preview()
        }
```